### ISSUE TITLE
Angular 4.x - Template parse warnings

### ISSUE THREAD

**Issue by x1unix** [valor-software/ngx-bootstrap#1923]
**Angular version**: 4.0.1
**ngx-bootstrap version**: 1.6.6

Console output:
```
Template parse warnings:
The <template> element is deprecated. Use <ng-template> instead ("
[WARNING ->]<template [ngIf]="!isClosed">
  <div [class]="'alert alert-' + type" role="alert" [ngClass]="classes""): ng:///AlertModule/AlertComponent.html@1:0
```

**Comment by ctrl-brk** [comment]
For Ng4 use ngx-bootstrap@next

**Comment by Martin-Luft** [comment]
For Angular 4 just use the @next version which fixes this warning. This cannot be fixed in 1.6.X because it would break Angular 2 support. You can use `"ngx-bootstrap": "2.0.0-beta.2"` or `npm install ngx-bootstrap@next`
